station_performance_pipeline/extract.py

```diff
@@ -0,0 +1,89 @@
+from os import environ as ENV
+from datetime import datetime
+
+import requests
+from requests.auth import HTTPBasicAuth
+from dotenv import load_dotenv
+
+
+def train_station_arrival_data(station_crs: str) -> list[dict]:
+    '''This function accepts the crs code of a station and returns a list of dictionaries.
+    Each dictionary represents a train that arrived at the station on this day.
+    The dictionary has the keys related to the expected and actual times of arrival and departure.'''
+
+    now = datetime.now()
+    response = requests.get(
```

Comment:
consider adding some error handling to the api call, e.g. raise exception if not 200 response

Reply:
Didn't consider that. Thanks!